scripts/tests/run_python_test.py

```diff
@@ -116,9 +116,10 @@ def forward_fifo(path: str, f_out: typing.BinaryIO, stop_event: threading.Event)
 @click.option("--quiet/--no-quiet", default=None,
               help="Do not print output from passing tests. Use this flag in CI to keep GitHub log size manageable.")
 @click.option("--load-from-env", default=None, help="YAML file that contains values for environment variables.")
+@click.argument("run", nargs=-1)
```

Comment:
> Should we have help with examples here?

arguments do not have help in click framework: https://click.palletsprojects.com/en/stable/arguments/

> also why did we pick this to be an argument instead

No particular reason TBH... Maybe I've seen similar API somewhere, not sure, though. I can change that to option. But IMHO as an argument it's easier to use it because it works like a pipeline somehow: `run-python-test.py <selected-script> | <run-filtering>`

> Even --script seems to be an option even though run_python_test would always require a script to run.

I think that everyone just copy paste boilerplate `click` code whiteout thinking how CLI should look like to be usable... The best example is `build_examples.py --target <TARGET> build` where fist of all target is required and secondly it is an option for `build` command not a global one :D Kind of madness when I've seen that for the first time. But with time I've got used to it...

Reply:
Changed to option